warp/tests/cuda/test_texture.py

```
@@ -2029,6 +2029,397 @@ def test_texture3d_array(test, device):
+        inputs=[tex, uvs, lods, output],
+        device=device,
+    )
+
+    result = output.numpy()
+    # Constant texture: all mip levels should have the same value
+    np.testing.assert_allclose(result, [value, value, value], rtol=1e-4, atol=1e-4)
+
+
+def test_mipmap2d_max_load(test, device):
+    """Test that sampling at the highest LOAD returns approximately the average of all texels."""
+    width, height = 4, 4
```

Ruff (0.14.14)

[error] Local variable `width` is assigned to but never used

Remove assignment to unused variable `width`

(F841)